tests/test_integrations/test__init__.py

```diff
@@ -54,13 +54,13 @@ def test_create_dataprovider(transfer):
                 conn_id="fivetran_default",
                 connector_id="filing_muppet",
                 group=Group(name="test_group"),
-                connector=Connector(
+                connector=FivetranConnector(
```

Comment:
What happens when we use a dict object instead of FivetranConnector object? is it working with it?

Reply:
This is taken care of in _init__ of the `FivetranIntegrations` class. This test here is used to fetch the transfer integrations class. In this scenario, the dict is already resolved to nested `FiveTranOptions` class so it should be as it is.